tests/unit_tests/test_checker.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+
+from app import app
+from app.version import APP_VERSION
+
+
+class CheckerTests(unittest.TestCase):
+
+    def setUp(self):
+        self.app = app.test_client()
+        self.assertEqual(app.debug, False)
+        self.route_prefix = "/v4/icons"
```

Comment:
Prefix could be taken from settings (see other comment above)

Reply:
done